readme.markdown

```diff
@@ -327,7 +327,7 @@ Aliases: `t.notLooseEqual()`, `t.notLooseEquals()`
 
 ## t.throws(fn, expected, msg)
 
-Assert that the function call `fn()` throws an exception. `expected`, if present, must be a `RegExp`, `Function`, or `Object`. The `RegExp` matches the string representation of the exception, as generated by `err.toString()`. For example, if you set `expected` to `/user/`, the test will pass only if the string representation of the exception contains the word `user`. Any other exception will result in a failed test. The `Function` is the exception thrown (e.g. `Error`). `Object` in this case corresponds to a so-called validation object, in which each property is tested for strict deep equality. As an example, see the following two tests--each passes a validation object to `t.throws()` as the second parameter. The first test will pass, because all property values in the actual error object are deeply strictly equal to the property values in the validation object.
+Assert that the function call `fn()` throws an exception. `expected`, if present, must be a `RegExp`, `Function`, or `Object`. The `RegExp` matches the string representation of the exception, as generated by `err.toString()`. For example, if you set `expected` to `/user/`, the test will pass only if the string representation of the exception contains the word `user`. Any other exception will result in a failed test. The `Function` will be called with the exception thrown (e.g. `Error`). `Object` in this case corresponds to a so-called validation object, in which each property is tested for strict deep equality. As an example, see the following two tests--each passes a validation object to `t.throws()` as the second parameter. The first test will pass, because all property values in the actual error object are deeply strictly equal to the property values in the validation object.
```

Comment:
Looking at https://github.com/ljharb/tape/blob/master/lib/test.js#L654, the `Function` passed to `expected` is actually called with error being thrown. This is what I have found most useful in my test cases.

And line 649~652 is about the error typing. And it is correctly described in the original statement. How about changing the sentence into "The `Function` is either constructor of the exception thrown (e.g. `Error`) or a plain function that to be called with that exception."

Reply:
ah, you're right - the logic is basically "if the error is an instanceof `expected`, it passes, otherwise if `expected` is an Error or Error subclass, it fails, otherwise it calls the function".

What do you suggest to convey that? An alternative would be to link to node's `assert.throws` docs rather than trying to replicate the prose.

Reply:
I'd reword the sentence making it clear in the first place: "`Function` could be the exception thrown, a validation function to be called with that exception or an error to fail the test. "

People is always in hurry, they will probably ignore (like myself) the line 367 😂

And PR commit is already updated.

Reply:
It isn't called with "that exception or an error", though.